src/assets/src/components/addQueue.tsx

```diff
@@ -130,6 +130,8 @@ export function AddQueuePage(props: PageProps) {
     const [descriptValidationResult, validateAndSetDescriptResult] = useStringValidation(queueDescriptSchema, true);
     const [allowedMeetingTypes, setAllowedMeetingTypes] = useState(new Set() as Set<string>);
     const [allowedValidationResult, validateAndSetAllowedResult] = useMeetingTypesValidation(props.backends);
+    const [physLocation, setPhysLocation] = useState('')
```

Comment:
Just as an example, it's fine and better that you don't use the underscore syntax here, since it's a local variable related to the form for this component.